pages/reference/supervisor/bandwidth-reduction.md

```diff
@@ -4,52 +4,41 @@ title: Reduce bandwidth usage
-| `BALENA_SUPERVISOR_VPN_CONTROL`        | true/false         | Enable / Disable cloudlink                         | true    | v1.1.0                            |
-| `BALENA_SUPERVISOR_CONNECTIVITY_CHECK` | true/false         | Enable / Disable cloudlink connectivity check      | true    | v1.3.0                            |
-| `BALENA_SUPERVISOR_POLL_INTERVAL`      | 600000 to 86400000 | API Poll interval in milliseconds                  | 900000  | v1.3.0                            |
-| `BALENA_SUPERVISOR_LOG_CONTROL`        | true/false         | Enable / Disable logs from being sent to API       | true    | v1.3.0                            |
-| `BALENA_SUPERVISOR_HARDWARE_METRICS`   | true/false         | Enable / Disable device hardware metrics reporting | true    | v12.8.0                           |
+**The minimum bandwidth required is approximately 1 MB per month**, when turning off all of the features below and setting API Polling to the minimum value of once per day. These measurements were made with a Quectel modem on a SixFab HAT on a Raspberry Pi4, and using a Twilio North American SIM. See this [blog post](https://www.balena.io/blog/cellular-iot-isnt-as-hard-as-you-think/) for a similar setup. The device was running balenaOS v2.107.10 and Supervisor v14.4.4.
 
-Side Effects / Warnings
--------------------
+| Control | Monthly Bandwidth if enabled | Variable Name<br>`BALENA_SUPERVISOR...` | Comments |
+| ------- | --------- | -------- | -------- |
+| Cloudlink VPN | 81 MB | ...`_VPN_CONTROL` | Bandwidth just to maintain connection |
+| Device Metrics | 73 MB | ...`_HARDWARE_METRICS` | Minimal application activity |
+| API Poll Interval, 4x hour | 28 MB | ...`_POLL_INTERVAL` | Default interval |
+| Network Manager connectivity check, 1x hour | 8 MB | No variable, see [configuration][config-json] to set interval | Default interval |
+| API Poll Interval, 1x day | 1 MB | ...`_POLL_INTERVAL` | Minimum interval |
+| Device Logging| ?? | ...`_LOG_CONTROL` | Minimal application activity |
+| Cloudlink connectivity check | N/A | ...`_CONNECTIVITY_CHECK`  | Only active when VPN is down |
 
-`BALENA_SUPERVISOR_VPN_CONTROL`: This defines the ability to send instantaneous updates to the device. Turning off cloudlink means that any configuration or variable update is reflected only when the device polls for these changes. The Web Terminal does not function when cloudlink is disabled. This also disables the public URL functionality.
+Each actual variable name is prefixed with "`BALENA_SUPERVISOR`", for example `BALENA_SUPERVISOR_VPN_CONTROL`.
 
-`BALENA_SUPERVISOR_CONNECTIVITY_CHECK`: Defines the device's ability to test and indicate (via an LED when available) that it has issues with connectivity.
+You can perform these measurements yourself as well! See the Network Metrics Logger [example](https://github.com/balena-io-examples/network-metrics-logger).
```

Comment:
Meta: Is it a good look for us to own the measurement tool to check our own metrics? It should be fine, if we can add a small paragraph on the "why we built this"? + If you can add alternatives that people can use to test but not as good as our checker for transparency. Showing the thinking process & what we used to build this will help the customers in the future who will be using this network logger to check their devices and find the results acceptable. 

Example: I love when my ISP can provide speed results using Ookla but I trust fast.com, ismyinternetgoodenough.com or at least some other tool to co-relate the performance.

Meta thought, feel free to skip

Reply:
Ultimately I think most/all tools report the statistics the networking subsystem provides, probably via `/proc`. Great idea to provide rationale. We build this to have a tool independent of the cellular provider. For example, Twilio does publish byte usage per device per hour, but you've got use its API. I did verify that our numbers match theirs really closely.

Reply:
SGTM, i think a rationale section be great to have